**Problems I faced:**
1. SignatureDoesNotMatch in generate_presigned_url.py while using generate_presigned_url
2. Unable to create a new resource/ update if a resource is deleted manually Example: S3 Bucket
3. Scan or Query in dynamodb
4. python mimetypes is not working for xlsx, docx extensions